### ISSUE TITLE
Available GPIOs aren't really documented. 

### ISSUE THREAD

**Issue by nicklasb** [Xinyuan-LilyGO/LilyGo-LoRa-Series#139]
I am trying to use the available pins on the T-Beam (1.1, but it seems most have the same docs), and while there are pinouts, it is not really completely usable, as pins are used for all the onboard stuff. And I don't think that there are any real official docs on that.  I would be really happy if I were wrong.

But for example, in my project (https://github.com/nicklasb/ShipRemote) to use ADC1 , I had to desolder the GPIO38 button and the connected pull down resistors, because I could not find any other pin for that. This as there are so few available GPIOs that I had to use a binary ladder to be able to add all the buttons i needed.

Next, I want it to communicate using TWAI, but again it is really hard to find available GPIOs, they are mostly taken in different ways. So it would be great if all _actually_ available GPIOs where listed so I won't have to try everyone again just to two hours later find that that it actually made the GPS or the AXP192 fail. 

Basically I am wondering it this is documented somewhere.

**Comment by lewisxhe** [comment]
There are not many available GPIOs left in T-Beam v1.x. For ADC, you can use GPIO36,39,

I have not used TWAI, but esp32 can freely map GPIO, so GPIO13 and 14 can be used as the GPIO of TWAI
![image](https://github.com/Xinyuan-LilyGO/LilyGo-LoRa-Series/assets/22990954/b87f6f4c-cd93-4b3d-a46a-c0353aed07fe)

**Comment by nicklasb** [comment]
Thanks for your response!
I have not been able to use the SensVN and SensVP-GPIOs like that. 
Then again, that might have was before I added a op amp (voltager follower). 36 and 39 may have had higher impedances for some reason, but I got really wierd values from them. If I remember correctly, it feels like I got some error using the ADC with those as well. But I'll retry at some time.

With regards to 13 and 14, thankt, I will try TWAI with them. 

(the general question is with regards to information like this in general somewhere, but I'll close it, egotistically :-) )
